pages/benchmark.tsx

```diff
@@ -0,0 +1,288 @@
+import React from 'react';
+import Head from 'next/head';
+import NextLink from 'next/link';
+import { Badge, Box, Container, Text, Link, Button } from '@modulz/design-system';
+import { StitchesLogo } from '../components/StitchesLogo';
+import { TitleAndMetaTags } from '../components/TitleAndMetaTags';
+import { format } from 'date-fns';
+
+function collectBenchesPerTestCase(entries) {
+  const map = new Map();
+  for (const entry of entries) {
+    const { commit, date, tool, benches } = entry;
+    for (const bench of benches) {
+      const result = { commit, date, tool, bench };
+      const arr = map.get(bench.name);
+      if (arr === undefined) {
+        map.set(bench.name, [result]);
+      } else {
+        arr.push(result);
+      }
+    }
+  }
+  return map;
+}
+
+const fontFamily = 'Untitled Sans, -apple-system, BlinkMacSystemFont, system-ui, sans-serif';
+
+export default function BenchmarkPage({ data, ...props }) {
+  if (!data) {
+    return null;
+  }
+
+  const dataSets = Object.keys(data.entries).map((name) => ({
+    name,
+    dataSet: collectBenchesPerTestCase(data.entries[name]),
+  }));
+
+  React.useEffect(() => {
+    // Global Chart.js overrides
+    (window as any).Chart.defaults.global.defaultFontFamily = fontFamily;
+    (window as any).Chart.defaults.global.animation.duration = 0;
+  }, []);
+
+  return (
+    <Box>
+      <Head>
+        <script src="https://cdn.jsdelivr.net/npm/chart.js@2.9.3/dist/Chart.min.js"></script>
```

Comment:
Hey @peduarte what about to use chart.css instead chart.js?

![image](https://user-images.githubusercontent.com/440220/112307480-c17f1b00-8c7f-11eb-912f-c2726dd9ca49.png)
![image](https://user-images.githubusercontent.com/440220/112307499-c8a62900-8c7f-11eb-9a6f-8913172951ae.png)

Reply:
Sounds like a good alternative. I'll check this out when I get back to this 🔥 thanks!